test/integration_new/conftest.py

```diff
@@ -0,0 +1,42 @@
+from test.integration_new import util
+import pytest
+
+
+@pytest.fixture(scope='package')
+def small_file_name():
+    return 'small.pdf'
+
+
+@pytest.fixture(scope='package')
+def small_file_path(small_file_name):
+    return util.get_file_path(small_file_name)
+
+
+@pytest.fixture(scope='package')
+def small_file_v2_name():
+    return 'small_v2.pdf'
+
+
+@pytest.fixture(scope='package')
+def small_file_v2_path(small_file_v2_name):
+    return util.get_file_path(small_file_v2_name)
+
+
+@pytest.fixture(scope='package')
```

Comment:
In .net and Java we are generating large file on the spot not to keep it in the source code. Maybe you can do the same. Why someone need to clone >20MB file just to run integration test?